index.html

```diff
@@ -129,55 +129,412 @@ <h2 data-dft-for="">
         "https://wicg.io">WICG</a>.
       </p>
     </section>
-    <section>
+    <section data-cite="DOM">
       <h2>
-        <code><dfn>note_taking</dfn></code> member
+        Installation prompts
       </h2>
       <p>
-        The <code>note_taking</code> member of the <a data-cite=
-        "appmanifest#web-application-manifest">Web Application Manifest</a> is
-        an <a data-cite="appmanifest#dfn-object">object</a> that contains
-        information related to note-taking. It has the following members:
+        There are multiple ways that the installation process can be triggered:
       </p>
       <ul>
-        <li>[=note_taking/new_note_url=]
+        <li>An end-user can <dfn data-lt="manual installation">manually</dfn>
+        trigger the installation process through the user agent's
+          <abbr title="User Interface">UI</abbr>, directly invoking the steps
+          to <a>present an install prompt</a>.
+        </li>
+        <li>The installation process can occur through a <dfn>site-triggered
+        install prompt</dfn>: the site can programmatically request that the
+        user agent present an install prompt to the user. The user agent MAY
+        restrict the availability of this feature to cases where, for instance,
+        there are sufficient <a data-cite="appmanifest#installability-signals">
+          installability signals</a> to warrant <a data-cite=
+          "appmanifest#dfn-installed">installation</a> of the web application.
         </li>
       </ul>
+      <p>
+        In any case, the user agent MUST NOT <a>present an install prompt</a>
+        if the document is not <a data-cite=
+        "appmanifest#dfn-is-installable">installable</a>.
```

Comment:
Maybe
```suggestion
        if the document is not [=installed web application|installable=].
```

Note that "is installable" is gone from the other spec... maybe it should be resurrected here?